coderd/database/migrations/000198_org_provisioners.up.sql

```diff
@@ -0,0 +1,14 @@
+-- At the time of this migration, only 1 org is expected in a deployment.
+-- In the future when multi-org is more common, there might be a use case
+-- to allow a provisioner to be associated with multiple orgs.
+ALTER TABLE provisioner_daemons
+	ADD COLUMN organization_id UUID REFERENCES organizations(id) ON DELETE CASCADE;
+
+UPDATE
+	provisioner_daemons
+SET
+	-- Default to the first org
+	organization_id = (SELECT id FROM organizations ORDER BY organizations.created_at ASC LIMIT 1 );
+
+ALTER TABLE provisioner_daemons
+	ALTER COLUMN organization_id SET NOT NULL;
```

Comment:
Do we still need to explicitly put this stuff in a transaction?  Otherwise a concurrent insert of a provisioner daemon could cause this `ALTER` to fail.

Reply:
No, by default all migrations run in a tx now.

https://github.com/coder/coder/pull/10966